### PR TITLE
PMM-12529 Run pmm-agent as non root

### PR DESCRIPTION
**Pull request by ademidoff**
PMM-12529

Link to the Feature Build: SUBMODULES-3472

### CODE REVIEW

File: managed/models/postgresql_version_test.go [hunk 1]
```diff
@@ -78,7 +78,7 @@ func TestGetPostgreSQLVersion(t *testing.T) {
 	for _, tc := range testCases {
 		tc := tc
 		t.Run(tc.name, func(t *testing.T) {
-			t.Parallel()
+			// t.Parallel()
```
Comment (artemgavrilov) [inline managed/models/postgresql_version_test.go:81]: Problematic test?

**Review by artemgavrilov — APPROVED** [review]

File: managed/models/postgresql_version_test.go (hunk 1 above)
Comment (ademidoff) [reply managed/models/postgresql_version_test.go]: Yes, permanently racing, so this was the only way to make them pass :(

**Review by BupycHuk — COMMENTED** [review]
Not sure if we need to make pmm2-client to be installed under pmm user for all installation. isn't it possible to do only for PMM Server?

**Comment by ademidoff** [comment]
> Not sure if we need to make pmm2-client to be installed under pmm user for all installation. isn't it possible to do only for PMM Server?

Maybe we can. Let me try :)

**Review by BupycHuk — APPROVED** [review]
please fix merge conflicts

**Pull request merged**